### DISCUSSION TITLE
Rückmeldung 6690 läuft mit Freetz

### DISCUSSION THREAD

**Discussion by Conan179** [Freetz-NG/freetz-ng#878] [Allgemein] [answered]
Hallo erst mal., in der make menuconfig steht:
- this device was never ever tested due to lack of hardware, please report if it works for you
 
Bei der 6690.
Ich kann die Rückmeldung geben, die 6690 läuft mit Freetz.

**Comment by fda77** [comment]
6690?

Die Meldung kommt von https://github.com/Freetz-NG/freetz-ng/blob/master/config/ui/firmware.in#L831
dh wenn FREETZ_TYPE_FIRMWARE_UNTESTED gesetzt ist

Dieses Symbol kommt von https://github.com/Freetz-NG/freetz-ng/blob/master/config/ui/firmware.in#L851
dort steht aber kein 6690

Könnte es sein dass du ein massiv verpatschtes Freetz nutzt? Zb diesen imatik quatsch von sammy? Dann müsstest du das dort melden

**Reply by Conan179** [reply]
Ja 6690, die kabel fritte.

öhm, nein, ich hab folgendes gemacht:
mkdir 6690
git clone https://github.com/Freetz-NG/freetz-ng 6690
cd 6690
make menuconfig
![6690_freetz_301023a](https://github.com/Freetz-NG/freetz-ng/assets/1906242/3fb09871-b156-43b7-8fb1-0f9e153c4884)
make

die .config hänge ich an
[.config.txt](https://github.com/Freetz-NG/freetz-ng/files/13205971/default.config.txt)

**Comment by fda77** [comment, marked as answer]
Ich hab es nochmal mit den Befehlen von dir ausprobieret (ohne "mkdir", das braucht man nicht) und was mit zuerst auffällt ist dass bei dir 385 Commits fehlen und danach dass dein Checkout 2 Monate alt ist.


![image](https://github.com/Freetz-NG/freetz-ng/assets/471850/31a7e3c8-6729-4702-994a-3de377ab846a)

**Reply by fda77** [reply]
Hier sind ein paar grundlegende git-Befehle, zb updaten und lokale Änderungen verwerfen: https://github.com/Freetz-NG/freetz-ng/blob/master/README.md#readme

**Reply by Conan179** [reply]
huch, es ist so alt schon? Dachte das war vor kurzem erst... git pull gemacht, Hinweis ist weg.
Tschuldigung

**Reply by Conan179** [reply]
mkdir nutzte ich deswegen, weil ich für meine fritten, jeweils einen ordner mache, momentan 7390,7272,6590,6591,6690.

**Reply by fda77** [reply]
Mache ich auch so. Nur kein mkdir :)
